volume-providers/vpc/vpcclient/instances/attach_volume.go

```diff
@@ -23,20 +23,22 @@ func (vs *VolumeAttachService) AttachVolume(volumeAttachmentTemplate *models.Vol
 	defer util.TimeTracker("AttachVolume", time.Now())
 
 	operation := &client.Operation{
-		Name:        "AttachVolume",
-		Method:      "POST",
-		PathPattern: vs.pathPrefix + instanceIDvolumeAttachmentPath,
+		Name:   "AttachVolume",
+		Method: "POST",
 	}
+	operation.PathPattern = vs.pathPrefix + instanceIDvolumeAttachmentPath
```

Comment:
previous format looks good, is there any difference else please use the previous format

Reply:
Addressed